### ISSUE TITLE
Operator upgrade broken

### ISSUE THREAD

**Issue by davidkarlsen** [redhat-developer/gitops-operator#292]
**Describe the bug**
When upgrading to 1.4.4 it fails on

```
docker pull registry.redhat.io/openshift-gitops-1/argocd-rhel8@sha256:ff4ad30752cf0d321cd6c2c6fd4490b716607ea2960558347440f2f370a586a8
Error response from daemon: error parsing HTTP 404 response body: invalid character '<' looking for beginning of value: "<HTML><HEAD><TITLE>Error</TITLE></HEAD><BODY>\nAn error occurred while processing your request.<p>\nReference&#32;&#35;132&#46;b65cda17&#46;1648076546&#46;3640bb61\n</BODY></HTML>\n"
```

**To Reproduce**
1. Upgrade operator to 1.4.4
2. observe your argos failing

**Expected behavior**
Operator should use images which in fact exist

**Screenshots**
N/A

**Additional context**
N/A

**Comment by iam-veeramalla** [comment]
Thanks @davidkarlsen for the reporting the issue.

We are actively working on this issue. We are committed to fixing this asap.

**Comment by lgoral** [comment]
I'm facing the same issue, please provide a fix. thx

**Comment by jannfis** [comment]
While we're working on a fix, there is a workaround described here: https://access.redhat.com/solutions/6839281

**Comment by julianhahn** [comment]
Any progess on this so far?
We are currently in the testing phase of an automatically setup openshift cluster. 
The manual workaround ist no option for us..

**Comment by KoljaKrueckmeyer** [comment]
I am facing the same issue today. temporary workaround for me is to change the container images in the corresponding pods from  registry.redhat.io/openshift-gitops-1/argocdrhel8@sha256:ff4ad30752cf0d321cd6c2c6fd4490b716607ea2960558347440f2f370a586a8
        to registry.redhat.io/openshift-gitops-1/argocd-rhel8@sha256:6fa09055767c899ed972dc5049f2a72cd60142b213a0e18a1c1437336934a166

**Comment by nastacio** [comment]
> The problem with the workaround: If you replace the wrong images with the correct images, the operator will replace them back to the wrong ones.

I have applied the workaround to 2 different clusters and the operator did not replace them back to the wrong ones. I think only a new version of the operator (presumably with the fix) will touch the CSV.

**Comment by jannfis** [comment]
Sorry for the late update. We're working hard to get a fixed version 1.4.5 out the door ASAP.

**Comment by jannfis** [comment]
v1.4.5 should now be available from Operator Hub.

**Comment by davidkarlsen** [comment]
@jannfis confirmed to work - thanks!

**Comment by lionelmace** [comment]
Thanks. It works great in the managed OpenShift `4.9.21_1528` on IBM Cloud :-)

**Comment by jannfis** [comment]
I will close this issue now, since the original problem seems to be resolved by v1.4.5.
